README.md

```diff
@@ -2,3 +2,9 @@ mapeditor
 =========
 
 OpenTibia Map Editor written in Go
+
+Instalation:
+
+Follow [Go Getting Started](https://golang.org/doc/install) page.<br />
+Run go get github.com/otland/mapeditor<br />
```

Comment:
`<br />` is not appropriate in Markdown files, just use empty line in between if you absolutely want to force a new line.